autopilot/betweenness_centrality.go

```diff
@@ -206,10 +206,30 @@ func (bc *BetweennessCentrality) Refresh(graph ChannelGraph) error {
 		}
 	}
 
-	// Divide by two as the graph is undirected.
+	// Get min/max to be able to normalize
+	// centrality values between 0 and 1.
+	var min, max float64
+
+	if len(centrality) > 0 {
+		min = centrality[0]
+		max = centrality[0]
+
+		for i := 1; i < len(centrality); i++ {
+			if centrality[i] < min {
+				min = centrality[i]
+			} else if centrality[i] > max {
+				max = centrality[i]
+			}
+		}
+	}
+
+	// Normalization factor.
+	z := max - min
+
+	// Normalize and create result map.
 	bc.centrality = make(map[NodeID]float64)
 	for u, value := range centrality {
-		bc.centrality[cache.Nodes[u]] = value / 2.0
+		bc.centrality[cache.Nodes[u]] = (value - min) / z
```

Comment:
is it possible to divide by zero?

Reply:
Initially I thought it's not possible, but then realized that it is if there are no edges just lone nodes. Corrected, ptal.